lib/active_triples/persistence_strategies/parent_strategy.rb

```diff
@@ -35,37 +35,38 @@ def persisted?
     #
     # @see PersistenceStrategy#destroy
     def destroy
-      final_parent.delete(obj.statements)
+      final_parent.delete(source.statements)
 
       parent.statements.each do |statement|
         parent.delete_statement(statement) if
-          statement.subject == obj.rdf_subject || 
-          statement.object == obj.rdf_subject
+          statement.subject == source.rdf_subject || 
+          statement.object == source.rdf_subject
       end
 
-      super 
+      super { source.clear }
     end
 
     ##
-    # Clear out any old assertions in the repository about this node or statement
-    # thus preparing to receive the updated assertions.
+    # @abstract Clear out any old assertions in the datastore / repository 
```

Comment:
Gah, this was copied from the [documentation for the method](https://github.com/ActiveTriples/ActiveTriples/blob/develop/lib/active_triples/persistence_strategies/persistence_strategy.rb#L61-L62) in `PersistenceStrategy`. Not intentional.